hieradata/node/lsstcam-ccobwb.cp.lsst.org.yaml

```diff
@@ -0,0 +1,55 @@
+---
+nm::connections:
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.81/26,139.229.175.126"
+        dns: "139.229.160.53;139.229.160.54;139.229.160.55;"
+        dns-search: "cp.lsst.org;"
+        method: "manual"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  enp6s0:
+    content:
+      connection:
+        id: "enp6s0"
+        uuid: "da97db8e-6400-4cca-9334-71995eb2ac1d"
+        type: "ethernet"
+        interface-name: "enp6s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "169.254.5.10/24"
```

Comment:
What is connected to this interface?

Reply:
> What is connected to this interface?

There is some Keysight device on one of these. (I would "at" youtsumi, but I guess it doesn't work for people who are not member of this repo.)
Ref https://lsstc.slack.com/archives/C06V270ECBW/p1716826028801209

Reply:
Yes, this one is a direct connection to some Keysight something, ref
https://lsstc.slack.com/archives/C06V270ECBW/p1715976340596719?thread_ts=1715953274.383569&cid=C06V270ECBW
and replies